tools/commit_stroller_tool/utils/cfg.json

```diff
@@ -0,0 +1,29 @@
+{
+    "commonConfig" : {
+        "modeMap" : {
+            "checkOutput" : "CheckOutputMode"
+        },
+        "traversalMap" : {
+            "firstBadVersion" : "FirstBadVersion"
+        },
+        "commandList" : [
+            {"cmd" : "git checkout {commit}", "catchMsg" : "error", "path" : "{gitPath}"},
+            {"cmd" : "git submodule init", "path" : "{gitPath}"},
+            {"cmd" : "git submodule update --recursive", "path" : "{buildPath}"},
+            {"cmd" : "cmake ../", "catchMsg" : "CMake Error", "path" : "{buildPath}"},
+            {"cmd" : "make --jobs=4", "path" : "{buildPath}"}
```

Comment:
suggest usage of generic cmake interface instead of make:
```bash
cmake --build . --parallel 4
```

Reply:
Done

Reply:
Rolled back temporarily, because it causes ccache errors sporadically in the current variant.